NEWS.md

```diff
@@ -1,3 +1,11 @@
+# mlr3spatiotempcv 0.0.0.9005
+
+* Add `spDataType` and `addBG` parameter to ResamplingSpCVBuffer
+* Fix TaskRegrST
+* Fix ResamplingSpCVEnv and ResamplingRepeatedSpCVEnv
+* Add generic unit tests
```

Comment:
I'll do this automatically after merging PRs via [fledge](https://github.com/krlmlr/fledge). This save you updating NEWS.md if you denote all news-worthy items with `*` in the final merge commit.

Reply:
So we can remove this?

Reply:
Yes. Usually its better to do version bumps in master and not directly in PRs.

The approach with {fledge} I am doing here only applies to repos for which I am the only one bumping.

Reply:
Done